tests/cli/add/add.test.ts

```diff
@@ -28,3 +28,85 @@ test.skip("add command installs package and updates .npmrc", async () => {
   const npmrc = await Bun.file(join(tmpDir, ".npmrc")).text()
   expect(npmrc).toContain("@tsci:registry=https://npm.tscircuit.com")
 })
+
+test.skip("add command handles different component path formats", async () => {
```

Comment:
```suggestion
test("add command handles different component path formats", async () => {
```

Is this skipped for purpose or anything unrelated?